packages/block-library/src/query-no-results/index.php

```diff
@@ -21,22 +21,24 @@ function render_block_core_query_no_results( $attributes, $content, $block ) {
 
 	$page_key   = isset( $block->context['queryId'] ) ? 'query-' . $block->context['queryId'] . '-page' : 'query-page';
 	$page       = empty( $_GET[ $page_key ] ) ? 1 : (int) $_GET[ $page_key ];
```

Comment:
Just fixing CS alignments 👍 
```suggestion
	$page_key = isset( $block->context['queryId'] ) ? 'query-' . $block->context['queryId'] . '-page' : 'query-page';
	$page     = empty( $_GET[ $page_key ] ) ? 1 : (int) $_GET[ $page_key ];
```